notes/stories/literature/novel-various.st.txt

```diff
@@ -1,3 +1,73 @@
+novel: Journey to the Center of the Earth (1867)
+================================================
+
+:: Title
+Journey to the Center of the Earth
+
+:: Date
+1867
+
+:: Description
+Journey to the Center of the Earth (French: Voyage au centre de la Terre),
+also translated with the variant titles A Journey to the Centre of the Earth
+and A Journey into the Interior of the Earth), is a classic science fiction
+novel by Jules Verne. It was first published in French in 1864, then reissued
+in 1867 in a revised and expanded edition. Professor Otto Lidenbrock is the
+tale's central figure, an eccentric German scientist who believes there are
+volcanic tubes that reach to the very center of the earth. He, his nephew
+Axel, and their Icelandic guide Hans rappel into Iceland's celebrated inactive
+volcano Snæfellsjökull, then contend with many dangers, including cave-ins,
+subpolar tornadoes, an underground ocean, and living prehistoric creatures
+from the Mesozoic and Cenozoic eras. (The 1867 revised edition inserted
+additional prehistoric material in Chaps. 37–39.) Eventually the three
+explorers are spewed back to the surface by an active volcano, Stromboli,
+in southern Italy.
+
+:: Authors
+Jules Verne
+
+:: References
+https://en.wikipedia.org/wiki/Journey_to_the_Center_of_the_Earth
+
+:: Ratings
+5 <paul>
+
+:: Choice Themes
+travel into the Earth [The whole story is about a trio of intrepid explorers who trek down old lava tubes to ultimately reach the center of the Earth.]
```

Comment:
Strikes me that this theme applies also in https://www.themeontology.org/story.php?name=writing%3A%20Inferno%20(1320)

Reply:
Excellent! And I see Wikipedia agrees <https://en.wikipedia.org/wiki/Subterranean_fiction>. Shall I leave you the honor?

Reply:
Forgot to talk about this today. I wanted to discuss how it relates to the "journey into the underworld" theme we have in Inferno, before adding.

Reply:
Very good. Shall  we merge this story and take up the matter of what to do with Inferno over the weekend?